crates/storage/db/Cargo.toml

```diff
@@ -78,6 +78,7 @@ mdbx = ["reth-libmdbx"]
 bench = []
 arbitrary = ["reth-primitives/arbitrary", "reth-db-api/arbitrary"]
 optimism = []
+disable_lock = []
```

Comment:
if this feature enabled do we then need sysinfo dep?

opting out is a lot harder than opting in, but in this case we expect that this is only disabled for testing, so imo this is fine